tests/wallet/sync/test_wallet_sync.py

```diff
@@ -248,19 +258,25 @@ async def test_almost_recent(
         await time_out_assert(30, wallet.get_confirmed_balance, 10 * calculate_pool_reward(uint32(1000)))
 
 
+@pytest.mark.parametrize("use_delta_sync", [False, True])
```

Comment:
it might make more sense to move this into a fixture in `conftest.py`. Then, any test could just take this parameter and automatically be parametrized by it